### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by kylecalbert**
Current page
Screenshot 2023-11-29 at 13 24 39

Actual design
Screenshot 2023-11-29 at 13 25 08

### CODE REVIEW

File: src/Pages/Product.js
Comment (sukhvir-notta) [inline src/Pages/Product.js:1]: The naming should be consistent across the project, `Pages` folder should be lowercase

File: src/Pages/Product.styled.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  display: grid;
+  /* background-color: red; */
```
Comment (sukhvir-notta) [inline src/Pages/Product.styled.js:5]: Clean up comments in this file if not needed

File: src/common/Text/Text.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import styled from 'styled-components';
+
+import { colors, fontWeight, sizes } from '../constants/constants';
+
+const textStyles = {
+  body: {
+    fontSize: sizes.extraSmall,
+    color: colors.darkGrayishBlue,
+  },
+  Headline: {
```
Comment (sukhvir-notta) [inline src/common/Text/Text.js]: Keep naming consistent with all `textStyles`. Should this be lowercase?

File: src/components/AddToCartButton.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import { StyledButton, IconWrapper } from './AddToCartButton.styled';
+
+const AddToCartButton = () => {
+  return (
+    <StyledButton>
+      <IconWrapper>
+        <FaCartPlus />
+      </IconWrapper>
+
+      <Text> Add to Cart</Text>
```
Comment (sukhvir-notta) [inline src/components/AddToCartButton.js]: Is the styling being applied to this text component?

File: src/Pages/Product.styled.js (hunk 1 above)
Comment (kylecalbert) [reply src/Pages/Product.styled.js:5]: i will be using them for future when styling the responsiveness

**Pull request merged**